ffmpeg/ffmpeg_test.go

```diff
@@ -1869,3 +1869,53 @@ func TestResolution_Clamp(t *testing.T) {
 	checkError = require.Error
 	test(l, Size{300, 600}, portrait, Size{300, 600})
 }
+
+func TestDurationFPS_GetCodecInfo(t *testing.T) {
+	run, dir := setupTest(t)
+	defer os.RemoveAll(dir)
+
+	//Generate test files
+	cmd := `
+	cp "$1/../data/duplicate-audio-dts.ts" test.ts
+	ffprobe -loglevel warning -show_format test.ts | grep duration=
+	ffprobe -loglevel warning -show_streams -select_streams v test.ts | grep r_frame_rate=
+
+	cp "$1/../data/bunny.mp4" test.mp4
+	ffmpeg -loglevel warning -i test.mp4 -c:v copy -t 2 -c:a copy -t 2.12564 test-short.mp4
```

Comment:
`-t` repeated twice here, which one should be used?

Reply:
Updated to have it only one